src/main/kotlin/extension/RedmadrobotExtension.kt

```diff
@@ -29,9 +30,16 @@ open class RedmadrobotExtension(objects: ObjectFactory) {
     /** Settings for android modules. */
     val android: AndroidSettings = AndroidSettings()
```

Comment:
Rename AndroidSettings to AndroidOptions, please :)
To make it consistent with TestOptions